build_and_install_all.sh

```diff
@@ -62,9 +62,7 @@ build() {
     rm -f "./$1/"*.pkg.tar.xz "./$1/"*.pkg.tar.xz.sig
     rm -f "./$1/"*.pkg.tar.zst "./$1/"*.pkg.tar.zst.sig
     # When building in a container, systemd's tests fail because of default Seccomp filters
-    if [ "$1" = 'systemd-selinux' ] && \
-        (systemd-detect-virt -q --container || systemd-detect-virt -q --private-users ) && \
-        grep '^Seccomp:\s*2$' /proc/self/status > /dev/null
+    if [ "$1" = 'systemd-selinux' ] && grep '^Seccomp:\s*2$' /proc/self/status > /dev/null
```

Comment:
It is sad, but it is known bug in systemd: https://github.com/systemd/systemd/issues/15393

As systemd's developers consider it to be a feature request and not a bug (despite "docker" being listed in the man page, https://www.freedesktop.org/software/systemd/man/systemd-detect-virt.html), I have to agree with this change.